### PR TITLE
Add basic functionality for displaying movie in new route on card click

### PR DESCRIPTION
**Pull request by SamuelColeman**
Im just rendering a movie card on the correct route but we can create a new component or toggle a few hidden attributes to display more info if needed. I also just bumped the favorite button outside of my link tags to prevent it from routing to the movie page.

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -97,6 +98,14 @@ class App extends Component {
         <Route exact path='/login' render={() => <Form /> } />
         <Route exact path='/' render={() => <MoviesContainer selectMovie={this.selectMovie} className='movie_container' signOutUser={this.signOutUser} removeFavourite={this.removeFavourite} handleFavourite={this.handleFavourite}/> } />
         <Route exact path='/favorites' render={() => <FavouritesContainer handleFavourite={this.handleFavourite}/>} />
+        <Route exact path='/movies/:id' render={({ match }) => {
```
Comment (Asilo5) [inline src/Components/App/App.js]: This ({ match }) looks interesting, haven't seen it before i don't think. It'd be great if you could explain this to us! If it works that is great!

File: src/Components/MovieCard/MovieCard.js [hunk 2]
```diff
@@ -14,13 +15,17 @@ const MovieCard = ({ title, poster_path, overview, id ,movies, currentUser, favo
         overview: currentMovie.overview
      }
         return (
-            <section className='movie_card' id={id}>
-                <h1>{ title }</h1>
-                <img className="movie__card--img" src={poster_path} alt={title} />
+           <section className='movie_card' id={id}> 
+                <Link to={`/movies/${id}`}>
```
Comment (Asilo5) [inline src/Components/MovieCard/MovieCard.js]: looks great!

File: src/Components/App/App.js (hunk 1 above)
Comment (Asilo5) [reply src/Components/App/App.js]: @SamuelColeman i think you can let me know through here what this does, before I merge it

**Comment by SamuelColeman** [comment]
I decided to add a new component for rendering the movie routes. I was having trouble using 'hidden' on the original movie card to display more info about the movie dynamically so I just made a new component. I still need to work on the styling but I think its a good start.

**Pull request merged**